src/com/puppetlabs/puppetdb/scf/storage.clj

```diff
@@ -1094,48 +1094,70 @@
     (dissoc row-map :environment_id)
     row-map))
 
+(defn normalize-resource-event
```

Comment:
Ah, I didn't think through to the different inputs that to-timestamp accepts. That makes sense.